cmd/kms-rest/startcmd/start.go

```diff
@@ -111,27 +111,27 @@ const (
 	kmsMasterKeyPathFlagUsage = "The path to the file with master key to be used for secret lock. If missing noop " +
 		"service lock is used. " + commonEnvVarUsageText + kmsMasterKeyPathEnvKey
 
-	operationalKMSStorageTypeFlagName  = "operational-kms-storage-type"
-	operationalKMSStorageTypeEnvKey    = "KMS_OPERATIONAL_KMS_STORAGE_TYPE"
-	operationalKMSStorageTypeFlagUsage = "The type of storage to use for Operational (user-specific) KMS. " +
-		"Supported options: mem, couchdb, sds. " + commonEnvVarUsageText + operationalKMSStorageTypeEnvKey
+	userKeyManagerStorageTypeFlagName  = "user-key-manager-storage-type"
```

Comment:
what does the prefix "user" imply here?

Reply:
yes, we have 2 of them:
* with lifetime of server - for supporting Keystore operations
* with lifetime of the request from user - for supporting user's kms/crypto operations